docs/releasing.md

```
@@ -156,3 +175,342 @@ At the beggining of `README.md` there's a line:
 
 Update the link on this line, i.e. replace, for example, `0.5.0` by `0.6.0rc1`
 
+## Konflux Release organization
+
+This section describes how Konflux **Applications**, **ReleasePlans**, **components**, **ReleasePlanAdmission (RPA)**, the Comet (catalog) container images, and release practice fit together, and how to perform **micro** and **minor** releases.
+
+
+#### Conventions
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)